### ISSUE TITLE
where there aren't titles, bold related text in paragraph

### ISSUE THREAD

**Issue by melteng** [DataUSA/datausa-site#427]
where visualizations don't have titles, bold the part of the paragraph that describes the viz context (or highlight in some other way). Example: the bar chart below doesn't have a title describing why it's showing these 5 occupations; the "5 most common occupations" text is easily lost.

<img width="1088" alt="screenshot 2016-08-25 17 33 39" src="https://cloud.githubusercontent.com/assets/8160651/17986719/14c2195a-6aea-11e6-9aa3-2fd57b1eaf19.png">

**Comment by davelandry** [comment]
I guess this begs the question, should we just change the title to something like "Race & Ethnicity by Wage for the Most Common Occupations" or is that too wordy?

**Comment by melteng** [comment]
I think changing the title would be the most honest and upfront (though I like how minimal it is).

How about change the title to "Race & Ethnicity by Wage for Common Jobs" ?
Same for "Gender by Wage for Common Jobs".

**Comment by davelandry** [comment]
Another added benefit of better titles (rather than bolding the text) is that they'll appear on embedded visualizations that don't have the paragraph text!
